readme.md

```diff
@@ -456,6 +456,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Pokémon](https://github.com/tobiasbueschel/awesome-pokemon) - Pokémon and Pokémon GO resources and tools.
 - [ChatOps](https://github.com/exAspArk/awesome-chatops) - Managing technical and business operations through a chat.
 - [Falsehood](https://github.com/kdeldycke/awesome-falsehood) - Falsehoods programmers believe in.
+- [FRC](https://github.com/andrewda/awesome-frc) - Resources for the First Robotics Competition.
```

Comment:
Use the full title here and the description should be what FRC is, not what it means.